0.5/articles/unit-testing-elements.md

```diff
@@ -378,7 +378,7 @@ We'll go with the setup/teardown approach for this example, by adding a new suit
 
     });
 
-So the first thing you may notice is our use `flush()`.  As we covered earlier, `flush` allows us to asynchronously dirty check pending objects are observed and ensures notification callbacks are dispatched accordingly. A synchronous alternative is `[element].deliverChanges()`, but that only works for observers declared directly on the element.
+So the first thing you may notice is our use of `flush()`.  As we covered earlier, `flush` allows us to asynchronously dirty check pending objects are observed (what??) and ensures notification callbacks are dispatched accordingly. A synchronous alternative is `[element].deliverChanges()`, but that only works for observers declared directly on the element.
```

Comment:
Re "(what??)" - I can't understand that sentence at all, so I can't even suggest a clearer way to say it.